src/nplinker/genomics/gcf.py

```diff
@@ -13,7 +12,7 @@
 
 class GCF():
 
-    def __init__(self, gcf_id: str) -> None:
+    def __init__(self, gcf_id: str, /) -> None:
```

Comment:
Can you explain what this does?  the `/` in the arguments?

Reply:
Parameters before “/” are positional-only parameters (see [PEP570](https://peps.python.org/pep-0570/#positional-only-parameters)). 
In this case, users must use `gcf = GCF("f001")` and cannot use `gcf = GCF(gcf_id="f001")`. So later we can easily rename `gcf_id` without bothering users.

The "/" is also used when the name of parameter does not really matter:

![image](https://user-images.githubusercontent.com/9798985/232054373-d8533880-2dd1-4ee7-996c-33b78059d877.png)